### PR TITLE
using new version of housekeeping that adds extra columns to the HK db

### PR DESCRIPTION
**Pull request by patduin**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -61,7 +61,7 @@
     <derby.version>10.10.2.0</derby.version>
     <beeju.version>1.2.1</beeju.version>
     <hamcrest-library.version>1.3</hamcrest-library.version>
-    <housekeeping.version>2.0.0</housekeeping.version>
+    <housekeeping.version>2.1.1-SNAPSHOT</housekeeping.version>
```
Comment (patduin) [inline pom.xml]: I suggest we bump to 2.2.0 release for HK?

**Comment by patduin** [comment]
> Add a line to the changelog describing the change.

I did already

**Comment by massdosage** [comment]
>  I did already

Ah sorry, I was only viewing a subset of changes.

File: circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/PartitionedTableReplication.java [hunk 2]
```diff
@@ -88,34 +88,38 @@ public void replicate() throws CircusTrainException {
       TableAndStatistics sourceTableAndStatistics = source.getTableAndStatistics(database, table);
       Table sourceTable = sourceTableAndStatistics.getTable();
 
-      PartitionsAndStatistics sourcePartitionsAndStatistics = source.getPartitions(sourceTable,
-          partitionPredicate.getPartitionPredicate(), partitionPredicate.getPartitionPredicateLimit());
+      PartitionsAndStatistics sourcePartitionsAndStatistics = source
```
Comment (massdosage) [inline circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/PartitionedTableReplication.java]: Has anything actually changed in this file other than the formatting?

Comment (patduin) [reply circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/PartitionedTableReplication.java]: line 106, adding db/table to the arguments

Comment (massdosage) [reply circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/PartitionedTableReplication.java]: OK

File: pom.xml (hunk 1 above)
Comment (massdosage) [reply pom.xml]: Maybe even 3.0.0, see my comments over there.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
-### 13.0.0 - 2018-10-15
+## TBD
+### Changed
+* Upgraded Housekeeping library to also store replica database and table name in Housekeeping database. See[#30](https://github.com/HotelsDotCom/housekeeping/issues/30).
```
[the remark this answers is not in the document]
Comment (patduin) [reply CHANGELOG.md]: yeah of course

File: pom.xml (hunk 1 above)
Comment (patduin) [reply pom.xml]: ack lets go for 3.0.0

**Comment by patduin** [comment]
fixes #88

**Pull request merged**